packages/ensnode-utils/src/constants.ts

```diff
@@ -0,0 +1,12 @@
+import { namehash } from "viem";
+import { Node } from "./types";
+
+export const ROOT_NODE: Node = namehash("");
+
+/**
+ * A set of nodes whose children are used for reverse resolution.
+ *
+ * Useful for identifying if a domain is used for reverse resolution.
+ * See apps/ensindexer/src/handlers/Registry.ts for context.
+ */
+export const REVERSE_ROOT_NODES: Set<Node> = new Set([namehash("addr.reverse")]);
```

Comment:
This will enable healing reverse addresses for both, `eth` plugin, and `linea.eth` plugin.

The goal of this PR was to only enable healing process for `eth` plugin.

For context:
- https://github.com/Consensys/linea-ens/issues/287

Reply:
Logged with:
- https://github.com/namehash/ensnode/issues/616